allure-pytest/src/allure/utils.py

```diff
@@ -84,8 +84,8 @@ def allure_full_name(nodeid):
 
 
 def step_parameters(func, *a, **kw):
-    all_names = inspect.getargspec(func).args
-    defaults = inspect.getargspec(func).defaults
+    all_names = inspect.getfullargspec(func).args
```

Comment:
Thank you a lot! But we still support python2.7. Can you add some logic here? For example, keep old method for old python with 'Six' library or some more smarter =)